[2025-05-23T15:56:27.632+0000] {processor.py:186} INFO - Started process (PID=63) to work on /opt/airflow/dags/dag.py
[2025-05-23T15:56:27.633+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-05-23T15:56:27.636+0000] {logging_mixin.py:190} INFO - [2025-05-23T15:56:27.636+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-05-23T15:56:27.658+0000] {logging_mixin.py:190} INFO - [2025-05-23T15:56:27.654+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 3, in <module>
    from airflow.providers.papermill.operators.papermill import PapermillOperator
ModuleNotFoundError: No module named 'airflow.providers.papermill'
[2025-05-23T15:56:27.660+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-05-23T15:56:27.685+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.062 seconds
[2025-05-23T15:56:58.318+0000] {processor.py:186} INFO - Started process (PID=127) to work on /opt/airflow/dags/dag.py
[2025-05-23T15:56:58.319+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-05-23T15:56:58.321+0000] {logging_mixin.py:190} INFO - [2025-05-23T15:56:58.321+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-05-23T15:56:58.332+0000] {logging_mixin.py:190} INFO - [2025-05-23T15:56:58.330+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 3, in <module>
    from airflow.providers.papermill.operators.papermill import PapermillOperator
ModuleNotFoundError: No module named 'airflow.providers.papermill'
[2025-05-23T15:56:58.333+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-05-23T15:56:58.353+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.039 seconds
[2025-05-23T15:57:29.207+0000] {processor.py:186} INFO - Started process (PID=191) to work on /opt/airflow/dags/dag.py
[2025-05-23T15:57:29.208+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-05-23T15:57:29.210+0000] {logging_mixin.py:190} INFO - [2025-05-23T15:57:29.210+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-05-23T15:57:29.223+0000] {logging_mixin.py:190} INFO - [2025-05-23T15:57:29.220+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 3, in <module>
    from airflow.providers.papermill.operators.papermill import PapermillOperator
ModuleNotFoundError: No module named 'airflow.providers.papermill'
[2025-05-23T15:57:29.224+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-05-23T15:57:29.240+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.037 seconds
[2025-05-23T15:57:59.886+0000] {processor.py:186} INFO - Started process (PID=255) to work on /opt/airflow/dags/dag.py
[2025-05-23T15:57:59.887+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-05-23T15:57:59.888+0000] {logging_mixin.py:190} INFO - [2025-05-23T15:57:59.888+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-05-23T15:57:59.907+0000] {logging_mixin.py:190} INFO - [2025-05-23T15:57:59.904+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 3, in <module>
    from airflow.providers.papermill.operators.papermill import PapermillOperator
ModuleNotFoundError: No module named 'airflow.providers.papermill'
[2025-05-23T15:57:59.908+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-05-23T15:57:59.922+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.041 seconds
[2025-05-23T15:58:30.897+0000] {processor.py:186} INFO - Started process (PID=319) to work on /opt/airflow/dags/dag.py
[2025-05-23T15:58:30.898+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-05-23T15:58:30.900+0000] {logging_mixin.py:190} INFO - [2025-05-23T15:58:30.899+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-05-23T15:58:30.911+0000] {logging_mixin.py:190} INFO - [2025-05-23T15:58:30.908+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 3, in <module>
    from airflow.providers.papermill.operators.papermill import PapermillOperator
ModuleNotFoundError: No module named 'airflow.providers.papermill'
[2025-05-23T15:58:30.912+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-05-23T15:58:30.927+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.035 seconds
[2025-05-23T15:59:01.664+0000] {processor.py:186} INFO - Started process (PID=389) to work on /opt/airflow/dags/dag.py
[2025-05-23T15:59:01.665+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-05-23T15:59:01.668+0000] {logging_mixin.py:190} INFO - [2025-05-23T15:59:01.667+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-05-23T15:59:01.681+0000] {logging_mixin.py:190} INFO - [2025-05-23T15:59:01.678+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 3, in <module>
    from airflow.providers.papermill.operators.papermill import PapermillOperator
ModuleNotFoundError: No module named 'airflow.providers.papermill'
[2025-05-23T15:59:01.682+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-05-23T15:59:01.701+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.042 seconds
[2025-05-23T15:59:32.505+0000] {processor.py:186} INFO - Started process (PID=453) to work on /opt/airflow/dags/dag.py
[2025-05-23T15:59:32.506+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-05-23T15:59:32.508+0000] {logging_mixin.py:190} INFO - [2025-05-23T15:59:32.508+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-05-23T15:59:32.520+0000] {logging_mixin.py:190} INFO - [2025-05-23T15:59:32.517+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 3, in <module>
    from airflow.providers.papermill.operators.papermill import PapermillOperator
ModuleNotFoundError: No module named 'airflow.providers.papermill'
[2025-05-23T15:59:32.522+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-05-23T15:59:32.538+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.037 seconds
[2025-05-23T16:00:02.606+0000] {processor.py:186} INFO - Started process (PID=517) to work on /opt/airflow/dags/dag.py
[2025-05-23T16:00:02.607+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-05-23T16:00:02.610+0000] {logging_mixin.py:190} INFO - [2025-05-23T16:00:02.609+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-05-23T16:00:02.625+0000] {logging_mixin.py:190} INFO - [2025-05-23T16:00:02.621+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 3, in <module>
    from airflow.providers.papermill.operators.papermill import PapermillOperator
ModuleNotFoundError: No module named 'airflow.providers.papermill'
[2025-05-23T16:00:02.626+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-05-23T16:00:02.645+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.046 seconds
[2025-05-23T16:00:33.191+0000] {processor.py:186} INFO - Started process (PID=581) to work on /opt/airflow/dags/dag.py
[2025-05-23T16:00:33.192+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-05-23T16:00:33.194+0000] {logging_mixin.py:190} INFO - [2025-05-23T16:00:33.194+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-05-23T16:00:33.208+0000] {logging_mixin.py:190} INFO - [2025-05-23T16:00:33.204+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 3, in <module>
    from airflow.providers.papermill.operators.papermill import PapermillOperator
ModuleNotFoundError: No module named 'airflow.providers.papermill'
[2025-05-23T16:00:33.209+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-05-23T16:00:33.225+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.040 seconds
